Marlin/src/pins/ramps/pins_FYSETC_F6_13.h

```diff
@@ -243,7 +244,6 @@
       #define LCD_BACKLIGHT_PIN               27
     #endif
 
-    #define KILL_PIN                          41
```

Comment:
This might look wrong, but it is already defined to this file higher in the file.